freqtrade/optimize/backtesting.py

```diff
@@ -127,8 +127,6 @@ def __init__(self, config: Dict[str, Any]) -> None:
         self.config['startup_candle_count'] = self.required_startup
         self.exchange.validate_required_startup_candles(self.required_startup, self.timeframe)
 
-        # TODO-lev: This should come from the configuration setting or better a
-        # TODO-lev: combination of config/strategy "use_shorts"(?) and "can_short" from the exchange
```

Comment:
https://github.com/freqtrade/freqtrade/pull/6397/files#r805210092

Reply:
I am honestly not convinced about the way for this.

do we really just ignore short files should we encounter them in a spot backtest/live run?